wiki/Modding/Bubble/vi.md

```diff
@@ -0,0 +1,55 @@
+---
+tags:
+ - bubble pop
+ - full bubble
+---
```

Comment:
```suggestion
no_native_review: true
---
```

Reply:
I think that this is done. But I will have to learn more about GitHub (just more, help.)